xilem/src/view/sized_box.rs

```diff
@@ -209,3 +221,84 @@ struct BorderStyle {
     width: f64,
     color: Color,
 }
+
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub struct Padding {
```

Comment:
Could this struct be moved into Masonry? I don't know a compelling argument for this to be Xilem specific.

Reply:
Yes, I've moved it.